Proofread Notification Endpoint section (#183)
openid/OpenID4VCI: openid-4-verifiable-credential-issuance-1_0.md

## Notification Error Response

> If the Notification Request does not contain an Access Token or contains an invalid Access Token, the Notification Endpoint returns an Authorization Error Response, as defined in Section 3 of [@!RFC6750].

Reviewer: the "such as" here was intentional so as not to preclude `WWW-Authenticate` challenges with schemes other than `Bearer` like `DPoP` 

```suggestion
If the Notification Request does not contain an Access Token or contains an invalid Access Token, the Notification Endpoint returns an Authorization Error Response, such as defined in Section 3 of [@!RFC6750].
```

When the `notification_id` value is invalid, the HTTP response MUST use the HTTP status code 400 (Bad Request) and set the content type to `application/json` with the following parameters in the JSON-encoded response body:

> * `error`: REQUIRED. A name at the top level of the object, the value of which SHOULD be one of the following ASCII [@!USASCII] error codes:

Reviewer 2: ```suggestion
* `error`: REQUIRED. The value of the `error` parameter SHOULD be one of the following ASCII [@!USASCII] error codes:
```
per PR #181.
